src/IdRegistry.sol

```diff
@@ -318,4 +317,26 @@ contract IdRegistry is TrustedCaller, Signatures, Pausable, EIP712, Nonces {
             sig
         );
     }
+
+    /*//////////////////////////////////////////////////////////////
+                             EXTERNAL VIEW HELPERS
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Verify a signature from a fid owner.
+     *
+     * @param signer   The address to check the signature of.
+     * @param fid      The fid to check the signature of.
+     * @param digest   The dgiest that was signed.
+     * @param sig      The signature to check.
+     */
+    function verifyFidSignature(
+        address signer,
```

Comment:
should call this `custody address` to match terminology elsewhere

Reply:
I meant that the `signer` param should be called `custodyAddress` instead. 

we should keep `verifyFidSignature` as is